processor/src/main/java/org/jboss/pnc/processor/ClientGenerator.java

```diff
@@ -206,7 +206,6 @@ private boolean process0(Set<? extends TypeElement> annotations, RoundEnvironmen
                         MethodSpec.Builder methodBuilder = beginMethod(restApiMethod);
                         addDefaultParameters(restApiMethod, methodBuilder);
                         methodBuilder
-                                .addException(ClassName.get("org.jboss.pnc.client", "ClientException"))
```

Comment:
Why did you remove the exception from the message signature ?

Reply:
Because this exception is not thrown anywhere.